backend/games/word_guesser/WordGuesser.py

```diff
@@ -0,0 +1,137 @@
+from random import randint
+from typing import Dict, List
+
+from String import isalpha
+
+MAX_GUESSES = 6
+WORD_LIST = [
+    "adieu",
+    "odium",
+    "shade",
+    "resin",
+    "alert",
+    "haunt",
+    "orate",
+    "media",
+    "blind",
+    "route",
+    "audio",
+    "pause",
+    "alien",
+    "canoe",
+    "plane",
+    "rouse",
+    "fraud",
+    "atone",
+    "raise",
+    "minor",
+]
+
+
+class WordGuesser:
+    def __init__(self):
+        # represents the state of the game, index 0 will hold the "goal word"
+        self.game_board: List[str] = [""]
+        self.guesses = {"guess1": "", "results": []}
+        self.goal_word = ""
+        self.guess_count = 0
+
+    # clears any previous activity and randomizes a word to initialize the list
+    def start_game(self):
+        # clear past game data
+        self.guesses = {"guess1": "", "results1": []}
+        self.game_board: List[str] = [""]
```

Comment:
This isn't really a game board, is it?

Reply:
It represents only the previous guesses and is meant to be used more internally than on the frontend